data/json/recipes/tools/tools_primitive.json

```diff
@@ -420,5 +420,33 @@
     "reversible": true,
     "autolearn": true,
     "components": [ [ [ "material_sand", 1 ] ], [ [ "leather", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "light_snare_kit",
+    "//": "Carving notches into the sticks (~15m), looping and knotting the string to sit securely in them (~30m), testing/misc/overhead (~5m)",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "difficulty": 3,
+    "decomp_learn": 1,
+    "//2": "Pop culture magazine lacks most helpful information, requires inferring and therefore doesn't learn the recipe until survival 2",
+    "book_learn": [
+      [ "mag_survival", 2 ],
+      [ "pocket_survival", 1 ],
+      [ "manual_survival", 1 ],
+      [ "textbook_survival", 1 ],
+      [ "survival_book", 1 ]
+    ],
+    "time": "50 m",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "proficiencies": [
+      { "proficiency": "prof_carving", "max_experience": "15 m" },
+      { "proficiency": "prof_fibers", "max_experience": "30 m" }
+    ],
```

Comment:
No Principles of trapping?

(Also Innawoods autolearn override plz ❤️)

Reply:
>No 5m Principles of trapping("testing/misc/overhead (~5m)")?

You already get that exp during setting/removing the trap, and the proficiency says it's for doing that (and only that).

>(Also Innawoods autolearn override plz ❤️)

What do you mean? It already autolearns at survival 3. Should be true in innawoods too.

Reply:
>What do you mean? It already autolearns at survival 3. Should be true in innawoods too.

I can't read :D

Reply:
>It already autolearns at survival 3.

I can't read :(